src/test/java/com/epam/lab/paymentsystem/controller/UserControllerTest.java

```diff
@@ -52,22 +46,12 @@
   @Autowired
   private RoleRepository roleRepository;
   private User user;
-  @Autowired
-  private H2TestDataInitializer h2TestDataInitializer;
-  private BCryptPasswordEncoder encoder;
 
   @BeforeEach
-  public void setUp() {
-    Authentication authentication = mock(Authentication.class);
-    SecurityContext securityContext = mock(SecurityContext.class);
-    when(securityContext.getAuthentication()).thenReturn(authentication);
-    SecurityContextHolder.setContext(securityContext);
-
-    h2TestDataInitializer.init();
-
+  public void setUpFields() {
     user = userRepository.getUserByLogin("test");
     when(userService.getCurrentUserLogin()).thenReturn(user.getLogin());
-    encoder = new BCryptPasswordEncoder();
+    new BCryptPasswordEncoder();
```

Comment:
No, you don't want to just leave creation of object without assignment. It is very unclear code